src/ignitus-StudentSignUp/Components/Email_Confirmed.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, {Component} from 'react'
+import progress from '../Images/Group 12.6.png';
+import newsletter from '../Images/newsletter_accept.png'
+import { Link } from 'react-router-dom';
+import { withErrorBoundary } from '../../ignitus-Internals';
+
+import '../Styles/style.scss';
+
+class Email_Confirmed extends Component {
+
+  render() {
+    return (
+      <div className="col-lg-12" style={{height: "55vw", backgroundColor: "white"}}>
```

Comment:
Please don't use `BootStrap 4 `as I have already mentioned that we are getting rid of it, and **migration to pure css**.